packages/devtools_app/lib/src/landing_screen.dart

```diff
@@ -147,8 +147,8 @@ class _LandingScreenBodyState extends State<LandingScreenBody> {
               Padding(
                 padding: const EdgeInsets.all(8.0),
```

Comment:
can you remove the left edge padding so it aligns with the edge of the text field?

Reply:
The padding was intentional to try to make the tip look more linked to the input field but perhaps it just looks ugly.

Reply:
<img width="478" alt="Screen Shot 2020-08-13 at 10 37 37 AM" src="https://user-images.githubusercontent.com/1226812/90167889-320ca380-dd51-11ea-8981-df445e8dd223.png">